packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/index.tsx

```diff
@@ -102,6 +102,8 @@ function ApplicationDetailsContent({ data, oidcConfig, onApplicationUpdated }: P
   };
 
   const onCloseDrawer = () => {
+    // The guide drawer may have updated the application data
+    onApplicationUpdated();
```

Comment:
this will only trigger the `mutate()` from `useSWR`, and it will not update the form which is expected since we do not have a central logic of handling such situation: what if the remote data updated when a form is open?

Reply:
I think it's good to keep what it is now. Since we don't have a mechanism to handle conflicts when different users are editing at the same time. I prefer to have each user edit independently, with the final submitted table data reflecting the most recent values.